aiida/manage/external/pgsu.py

```diff
@@ -13,344 +13,11 @@
-    conn.close()
-    return output
-
-
-def _execute_psql(command, user='postgres', quiet=True, interactive=False, **kwargs):
-    """
-    Executes an SQL command via ``psql`` as another system user in a subprocess.
-
-    Tries to "become" the user specified in ``kwargs`` (i.e. interpreted as UNIX system user)
-    and run psql in a subprocess.
-
-    :param command: A psql command line as a str
-    :param quiet: If True, don't print warnings.
-    :param interactive: If False, `sudo` won't ask for a password and fail if one is required.
-    :param kwargs: connection details to forward to psql, signature as in psycopg2.connect
-    """
-    option_str = ''
-
-    database = kwargs.pop('database', None)
-    if database:
-        option_str += '-d {}'.format(database)
-    # to do: Forward password to psql; ignore host only when the password is None.  # pylint: disable=fixme
-    kwargs.pop('password', None)
-
-    host = kwargs.pop('host', 'localhost')
-    if host and host != 'localhost':
-        option_str += ' -h {}'.format(host)
-    elif not quiet:
-        click.echo(
-            "Warning: Found host 'localhost' but dropping '-h localhost' option for psql " +
-            'since this may cause psql to switch to password-based authentication.'
-        )
-
-    port = kwargs.pop('port', None)
-    if port:
-        option_str += ' -p {}'.format(port)
-
-    user = kwargs.pop('user', 'postgres')
-
-    # Build command line
-    sudo_cmd = ['sudo']
-    if not interactive:
-        sudo_cmd += ['-n']
-    su_cmd = ['su', user, '-c']
-
-    psql_cmd = ['psql {opt} -tc {cmd}'.format(cmd=escape_for_bash(command), opt=option_str)]
-    sudo_su_psql = sudo_cmd + su_cmd + psql_cmd
-    result = subprocess.check_output(sudo_su_psql, **kwargs)
-    result = result.decode('utf-8').strip().split('\n')
-    result = [i for i in result if i]
-
-    return result
-
-
-def escape_for_bash(str_to_escape):
-    """
-    This function takes any string and escapes it in a way that
-    bash will interpret it as a single string.
-
-    Explanation:
-
-    At the end, in the return statement, the string is put within single
-    quotes. Therefore, the only thing that I have to escape in bash is the
-    single quote character. To do this, I substitute every single
-    quote ' with '"'"' which means:
-
-    First single quote: exit from the enclosing single quotes
-
-    Second, third and fourth character: "'" is a single quote character,
-    escaped by double quotes
-
-    Last single quote: reopen the single quote to continue the string
-
-    Finally, note that for python I have to enclose the string '"'"'
-    within triple quotes to make it work, getting finally: the complicated
-    string found below.
-    """
-    escaped_quotes = str_to_escape.replace("'", """'"'"'""")
-    return "'{}'".format(escaped_quotes)
+import warnings
+from pgsu import PGSU, PostgresConnectionMode, DEFAULT_DSN  # pylint: disable=unused-import
```

Comment:
Maybe also add an alias for `DEFAULT_DBINFO` to `DEFAULT_DSN` since the name has changed